src/components/Investigaciones/autorxs/Autorxs.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import styles from './Autorxs.module.css';
+
+const Autorxs = () => {
+  const [autorxsData, setAutorxsData] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch('../src/components/investigaciones/autorxs/autorxs.json'); // Ajusta la ruta según tu estructura
+        const data = await response.json();
+        setAutorxsData(data);
+      } catch (error) {
+        console.error('Error al cargar datos:', error);
+      }
+    };
```

Comment:
you don't need this,
just `import "autorxs.json"`
in the 'real life' we'll use DecapCMS and/or react router to actually load data.